cmd/src/code_intel_upload.go

```diff
@@ -207,14 +207,20 @@ func printInferredArguments(out *output.Output) {
 
 // makeCodeIntelUploadURL constructs a URL to the upload with the given internal identifier.
 // The base of the URL is constructed from the configured Sourcegraph instance.
-func makeCodeIntelUploadURL(uploadID int) (string, error) {
+func makeCodeIntelUploadURL(uploadID int, isSCIPAvailable bool) (string, error) {
 	url, err := url.Parse(cfg.Endpoint)
 	if err != nil {
 		return "", err
 	}
 
 	graphqlID := base64.URLEncoding.EncodeToString([]byte(fmt.Sprintf(`LSIFUpload:%d`, uploadID)))
-	url.Path = codeintelUploadFlags.repo + "/-/code-intelligence/uploads/" + graphqlID
+
+	codeGraphPart := "code-graph"
+	if !isSCIPAvailable {
```

Comment:
Was debating whether to do this at all. The code-graph URL changed before we added SCIP support but I figured this was a simple approximation